test/test_binary_ufuncs.py

```diff
@@ -1435,6 +1443,19 @@ def test_maximum_minimum_cross_device(self, device):
             self.assertEqual(tensor_result_1, numpy_result_1)
             self.assertEqual(tensor_result_2, numpy_result_2)
 
+    @dtypes(*product(get_all_fp_dtypes(), get_all_fp_dtypes()))
+    def test_maximum_and_minimum_subgradient(self, device, dtypes):
```

Comment:
In this case I guess the numerical Jacobian would be computing the same values that we desire, so I guess it is technically possible here to subsume.

But the flipside is that in the case of `relu`, you could have a situation where the desired min-norm subgradient is different from what is computed by finite differencing, so it might be easier to just say as a general rule, don't rely on gradcheck if we are testing subgradients

Also if someone wants to understand the behavior of the op at these non-differentiable points, reading the cases in this test would be easier than reasoning about how numerical differencing computes the values.

Reply:
OK, sounds good